x-pack/plugins/apm/ftr_e2e/cypress_test_runner.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import Url from 'url';
+import cypress from 'cypress';
+import { esTestConfig } from '@kbn/test';
+import { apm, createLogger, LogLevel } from '@kbn/apm-synthtrace';
+import path from 'path';
+import { FtrProviderContext } from './ftr_provider_context';
+import { createApmUsers } from '../scripts/create_apm_users/create_apm_users';
+
+export async function cypressTestRunner({ getService }: FtrProviderContext) {
+  const config = getService('config');
+  const kibanaVersion = esTestConfig.getVersion();
+
+  const kibanaUrl = Url.format({
+    protocol: config.get('servers.kibana.protocol'),
+    hostname: config.get('servers.kibana.hostname'),
+    port: config.get('servers.kibana.port'),
+  });
+
+  const username = config.get('servers.elasticsearch.username');
+  const password = config.get('servers.elasticsearch.password');
```

Comment:
What do you think of renaming it?
```suggestion
  const esUsername = config.get('servers.elasticsearch.username');
  const esPassword = config.get('servers.elasticsearch.password');
```